doc/latex.rst

```diff
@@ -65,6 +65,47 @@ avoid interpretation as escape sequences, or use raw strings (as is done here).
    }
    latex_show_urls = 'footnote'
 
+.. _latex-documentclass:
+
+The LaTeX document class
+------------------------
+
+LaTeX document classes are files (with extension ``.cls``) containing LaTeX
+macros, often devoted to custom layout definitions.  They are loaded via
+the ``\documentclass{<classname>}`` line in a LaTeX source file.
+
+Sphinx provides two LaTeX classes ``sphinxhowto`` and ``sphinxmanual`` of its
+own (the two ``.cls`` files are automatically copied over to the LaTeX build
+directory).  Which one is used depends on whether ``'howto'`` or ``'manual'``
+is encountered in the :confval:`latex_documents` variable (the default uses
+``'manual'``).  Any string distinct from these two is interpreted as the name
+of a LaTeX class which will be loaded in place of the Sphinx custom ones.  In
+this case, ``sphinx.sty`` will let ``\sphinxmaketitle``,
+``\sphinxtableofcontents``, ``sphinxthebibliography`` and ``sphinxtheindex``
+default to the LaTeX commands and environments without sphinx prefix.
+
+The Sphinx classes are slim wrappers around the standard LaTeX classes
+``article`` and ``report`` respectively (Japanese documents use ``jreport``
+and ``jsbook`` instead).  It is possible to alter the mapping between Sphinx
+and base classes via the :confval:`latex_docclass` dictionary.
+
+As explained above, ``'howto'`` and ``'manual'`` are special keywords, and in
+particular with ``'howto'`` the Sphinx LaTeX writer is told not to use the
+``\chapter`` mark-up (except for Japanese).  It is possible to combine their
+usage with custom classes.  This is done with the help of
+:confval:`latex_docclassprefix`.  If for example it is set to ``'my'`` then
+``myhowto.cls`` or ``mymanual.cls`` respectively will be loaded.  The user is
+free inside such a custom class to ``\LoadClass`` either ``sphinxhowto`` or
+``sphinxmanual``.  This allows to keep some of their custom settings and also
```

Comment:
…to ``\LoadClass`` either ``sphinxhowto`` or ``sphinxmanual`` in order to keep some of their custom settings and override others.